app/build.gradle

```diff
@@ -47,6 +48,15 @@ android {
 //        }
 //    }
 
+    buildTypes {
+        debug {
+        }
+        release {
+            minifyEnabled true
```

Comment:
Yeap, this shrink apk size and early prepare you to 65k limit. It's more complicated to add proguard to existing codebase than using proguard from beginning